frontend/src/app/app.component.html

```diff
@@ -2,4 +2,5 @@
 <div class="container">
   <router-outlet></router-outlet>
   <h3>It should be farily obvious this is a placeholder.</h3>
+  <app-settings></app-settings>
```

Comment:
if this is a display component, i think we probably want to put it on a profile page... that lol is sitting in a branch from several months ago. and probably obselete by this point and i should just start over but i'll go look at it before i decide that i guess